### ISSUE TITLE
Missing button on page

### ISSUE THREAD

**Issue by kevmc21** [18F/code-of-conduct#31]
I just noticed that on 'Accessibility for teams' there seems to be a button missing to move from the Information Architecture page to the Video and Multimedia page below the WCAG references.
